contracts/sfc/SFC.sol

```diff
@@ -264,17 +259,20 @@ contract SFC is Initializable, Ownable, Version {
         _syncValidator(validatorID, true);
     }
 
+    /// Set redirection authorizer for approving redirections
     function setRedirectionAuthorizer(address v) external onlyOwner {
         if (redirectionAuthorizer == v) {
             revert SameRedirectionAuthorizer();
         }
         redirectionAuthorizer = v;
     }
 
+    // Announce redirection of address to another address
```

Comment:
... To be called by validator whose auth key was compromised. Produced event serves as a proof of support request validity. Redirect authorizer then creates appropriate redirect proposal by calling initiateRedirection()